js/app/libs.js

```diff
@@ -137,18 +214,18 @@ Fliplet.Registry.set('notification-inbox:1.0:app:core', function(data) {
     ts = ts || Date.now();
     opt = opt || {};
 
-    return getNewNotifications(ts)
+    return getLatestNotificationCounts(ts, opt)
       .then(function(counts) {
         var data = {
           updatedAt: Date.now(),
           unreadCount: counts[1],
           newCount: counts[0]
         };
-        var comparisonProps = ['unreadCount', 'newCount'];
+        var comparisonProp = notificationsBadgeType === 'unread' ? 'unreadCount' : 'newCount';
```

Comment:
wouldn't it be simpler to have `notificationsBadgeType` default to `new` when you set the inital variable so you don't have multiple places needing to check for the default?

Reply:
Good idea. Committed update.